prodtests/full-system-test/aggregator-workflow.sh

```diff
@@ -195,7 +197,14 @@ if [[ $AGGREGATOR_TASKS == BARREL_TF ]] || [[ $AGGREGATOR_TASKS == ALL ]]; then
     : ${DELAYINTFS_MEANVTX:=10}
     add_W o2-calibration-mean-vertex-calibration-workflow "" "MeanVertexCalib.tfPerSlot=$TFPERSLOTS_MEANVTX;MeanVertexCalib.maxTFdelay=$DELAYINTFS_MEANVTX"
   fi
-
+  # ITS
+  if [[ $CALIB_ITS_DEADMAP_TIME == 1 ]]; then
+     add_W o2-itsmft-deadmap-builder-workflow
+  fi
+  # MFT
+  if [[ $CALIB_MFT_DEADMAP_TIME == 1 ]]; then
+     add_W o2-itsmft-deadmap-builder-workflow  "--runmft"
```

Comment:
@nicolovalle could you please check if some default options should be changed (e.g. `--skip-static-map`) ?

Reply:
Dear @shahor02 , thank you for this.

Indeed, we can add `—skip-static-map` for MFT, *not for ITS*.

The reason is that he static part of the map is by construction included in the time-evolving one for MFT but not for ITS.

Reply:
For all the other options the default are good as they are.